### ISSUE TITLE
Add hours does not work

### ISSUE THREAD

**Issue by t0cableguy** [WazeUSA/WME-Place-Harmonizer#7]
Add hours  not working, replace all fully functional

**Comment by mapomatic** [comment]
I'm going to try to tackle this one.  Will work on a branch off of Beta, then implement in Beta for testing, and finally copy to master.

**Comment by mapomatic** [comment]
I've narrowed the problem down to this line in the getOpeningHours function:
var formatOpeningHour = require('Waze/ViewHelpers').formatOpeningHour;

Unfortunately, Waze/ViewHelpers doesn't appear to be in the list of requires that Waze devs fixed for us, and I don't know how to get to the formatOpeningHour function without it.  I've asked for help in the WME Beta Testing forum: https://www.waze.com/forum/viewtopic.php?f=211&t=213641

Not sure if that will go anywhere though.  I'm going to move on to look at other bugs.  If nothing happens on this one for a while, I'll see if I can somehow reverse engineer what the formatOpeningHour function is doing and try to duplicate it.

**Comment by RavenDT** [comment]
This is the error that I get:

```
Uncaught Error: Cannot find module './Waze/ViewHelpers.js'.
    at https://www.waze.com/assets-editor/js/app-3db7e45c3520ee2c8b29.js:25:19369
    at s (https://www.waze.com/assets-editor/js/app-3db7e45c3520ee2c8b29.js:25:19410)
    at n (https://www.waze.com/assets-editor/js/app-3db7e45c3520ee2c8b29.js:25:19319)
    at window.require (https://www.waze.com/assets-editor/js/app-3db7e45c3520ee2c8b29.js:24:28842)
    at getOpeningHours (eval at <anonymous> (:2:297), <anonymous>:5360:37)
    at Object.action (eval at <anonymous> (:2:297), <anonymous>:2246:57)
    at HTMLInputElement.button.onclick (eval at <anonymous> (:2:297), <anonymous>:4998:25)
(anonymous) @ app-3db7e45….js:25
s @ app-3db7e45….js:25
n @ app-3db7e45….js:25
window.require @ app-3db7e45….js:24
getOpeningHours @ VM60:5360
action @ VM60:2246
button.onclick @ VM60:4998
```

**Comment by RavenDT** [comment]
Fix is in pull request #20

**Comment by t0cableguy** [comment]
console log, 2nd run of "add hours". If i add this set of hours first it parses fine.
run first. 
`WMEPH-Beta: sat-sun 10a-5p
VM81:7316 WMEPH-Beta: sat-sun 10a-5p,
VM81:7316 WMEPH-Beta: Initial parse: sat-sun 10a-5p 
VM81:7316 WMEPH-Beta: After kill terms: sat-sun 10a-5p 
VM81:7316 WMEPH-Beta: Replace day terms: SS-UU 10a-5p 
VM81:7316 WMEPH-Beta: Replace month terms: SS-UU 10a-5p 
VM81:7316 WMEPH-Beta: AMPM parse: SS-UU 10AA-5PP 
VM81:7316 WMEPH-Beta: MM/TT format: SS-UU 10AA-5PP 
VM81:7316 WMEPH-Beta: XX:XX format: SS-UU 10:00-17:00 
VM81:7316 WMEPH-Beta: Add dash: SS-UU 10:00-17:00 
VM81:7316 WMEPH-Beta: Split: SSUU,10:00-17:00
VM81:7316 WMEPH-Beta: {"fromHour":"10:00","toHour":"17:00","days":[0,6]}
VM81:7316 WMEPH-Beta: [{"fromHour":"10:00","toHour":"17:00","days":[0,6]}]
VM81:7316 WMEPH-Beta: Ran highlighter in 3.084999999991851 milliseconds.
VM81:7316 WMEPH-Beta: Building banners`

run second.
`WMEPH-Beta: sat-sun 10a-5p
VM81:7316 WMEPH-Beta: sat-sun 10a-5p,[object object]
VM81:7316 WMEPH-Beta: Initial parse: sat-sun 10a-5p  object object 
VM81:7316 WMEPH-Beta: After kill terms: sat-sun 10a-5p  object object 
VM81:7316 WMEPH-Beta: Replace day terms: SS-UU 10a-5p  object object 
VM81:7316 WMEPH-Beta: Replace month terms: SS-UU 10a-5p  object object 
VM81:7316 WMEPH-Beta: AMPM parse: SS-UU 10AA-5PP  object object 
VM81:7316 WMEPH-Beta: Extra words in the string
VM81:7508 WMEPH-Beta: Can't parse those hours
VM81:7316 WMEPH-Beta: Building banners`

**Comment by mapomatic** [comment]
Fixed in 1.1.50
